README.md

```diff
@@ -44,6 +44,39 @@ In order to configure some aspects of layout, like the header, edit `config/tech
 
 If you move pages around and URLs change, make sure you set up redirects from the old URLs to the new URLs.
 
+## Making changes to a diagram
+Edit the `drawio` files in `source/images/originals` using the draw.io desktop app.
```

Comment:
In the steps for editing a diagram the files are called draw.io so it would be good to be consistent.